devito/passes/clusters/aliases.py

```diff
@@ -211,9 +216,19 @@ def extract(cls, n, context, min_cost, max_alias, cluster, sregistry):
     def in_writeto(cls, max_par, dim, cluster):
         return PARALLEL in cluster.properties[dim]
 
+
+class CallbacksDivs(CallbacksInvariants):
+
+    mode = 'divs'
+
+    @classmethod
+    def _extract_rule(cls, context, min_cost, cluster):
+        return lambda e: (e.is_Pow and e.exp.is_Integer and e.exp < 0 and
+                          all(i.function.is_const for i in e.base.free_symbols))
+
     @classmethod
-    def selector(cls, min_cost, cost, naliases):
-        return cost >= min_cost and naliases >= 1
+    def selector(cls, min_cost, cost):
+        return int(cost > 0)
```

Comment:
not `> min_cost` ?

Reply:
no, I want all scalar divisions to be hoisted out, so `min_cost` is ignored. In fact, in `divs` mode, `min_cost` contains a dummy value (I think `None`)